release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1158,6 +1220,31 @@ In the following tables, features are marked with the following statuses:
 
 |====
 
+[discrete]
+=== OpenShift CLI (oc) Technology Preview features
+
+.OpenShift CLI (oc) Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.14 |4.15 |4.16
+
+|oc-mirror Plugin v2
```

Comment:
match other caps?
```suggestion
|oc-mirror plugin v2
```

Reply:
Will Enclave Support be Enclave support?

Reply:
Great. But I have noticed other features are capitalized.

Reply:
Like for example these- not available and technology preview and so on?
`|oc-mirror plugin v2
|Not Available
|Not Available
|Technology Preview`